NPM package postinstall script fails

Hello,

I am attempted to install jsxhint but cannot seem to complete the build process. Can you help me out?

```
> jsxhint@0.3.0 postinstall C:\Users\Matthew\AppData\Roaming\npm\node_modules\jsxhint
> if [[ -d .git ]]; then cp pre-commit.sh .git/hooks/pre-commit && chmod +x .git/hooks/pre-commit; fi

-d was unexpected at this time.
npm ERR! jsxhint@0.3.0 postinstall: `if [[ -d .git ]]; then cp pre-commit.sh .git/hooks/pre-commit && chmod +x .git/hooks/pre-commit; fi`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the jsxhint@0.3.0 postinstall script.
npm ERR! This is most likely a problem with the jsxhint package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     if [[ -d .git ]]; then cp pre-commit.sh .git/hooks/pre-commit && chmod +x .git/hooks/pre-commit; fi
npm ERR! You can get their info via:
npm ERR!     npm owner ls jsxhint
npm ERR! There is likely additional logging output above.

npm ERR! System Windows_NT 6.2.9200
npm ERR! command "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "JSXHint"
npm ERR! cwd C:\Users\Matthew
npm ERR! node -v v0.10.24
npm ERR! npm -v 1.3.21
npm ERR! code ELIFECYCLE
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     C:\Users\Matthew\npm-debug.log
npm ERR! not ok code 0
```

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
   "scripts": {
     "test": "tap test/test.js",
     "lint": "jshint",
-    "postinstall": "if [[ -d .git ]]; then cp pre-commit.sh .git/hooks/pre-commit && chmod +x .git/hooks/pre-commit; fi",
+    "postinstall": "node post-install",
     "precommit": "npm run lint && npm test",
     "prepublish": "docme README.md-- --configure ./.jsdocrc.json"
   },

--- a/post-install.js
+++ b/post-install.js
@@ -1,0 +1,16 @@
+'use strict';
+
+var fs = require('fs');
+var src = './pre-commit.sh';
+var out = './.git/hooks/pre-commit';
+
+fs.exists('./.git', function(yes){
+  if(yes){
+    fs.createReadStream(src)
+      .pipe(fs.createWriteStream(out)
+        .on('close', function(){
+          fs.chmod(out, '755');
+        })
+      );
+  }
+});

